testsuite/OMSimulator/import_export_signalFilter.lua

```diff
@@ -0,0 +1,201 @@
+-- status: correct
+-- teardown_command: rm -rf signalFilter-lua/
+-- linux: yes
+-- linux32: yes
+-- mingw: yes
+-- win: no
+-- mac: no
+
+function readFile(filename)
+  local f = assert(io.open(filename, "r"))
+  local content = f:read("*all")
+  print("\n")
+  print(content)
+  f:close()
+end
+
+oms_setCommandLineOption("--suppressPath=true --skipCSVHeader=true --addParametersToCSV=true")
+oms_setTempDirectory("./signalFilter-lua/")
+
+oms_newModel("model")
+
+oms_addSystem("model.root", oms_system_wc)
+
+oms_addSystem("model.root.System1", oms_system_sc)
+
+oms_addConnector("model.root.System1.Input_1", oms_causality_input, oms_signal_type_real)
+oms_addConnector("model.root.System1.parameter_1", oms_causality_parameter, oms_signal_type_real)
+
+oms_setReal("model.root.System1.Input_1", 10)
+oms_setReal("model.root.System1.parameter_1", 20)
+
+oms_addSystem("model.root.System2", oms_system_sc)
+
+oms_addConnector("model.root.System2.Input_1", oms_causality_input, oms_signal_type_real)
+oms_addConnector("model.root.System2.parameter_1", oms_causality_parameter, oms_signal_type_real)
+
+oms_setReal("model.root.System2.Input_1", 30)
+oms_setReal("model.root.System2.parameter_1", 40)
+
+oms_addSubModel("model.root.Gain", "../resources/Modelica.Blocks.Math.Gain.fmu")
+
+oms_addSignalsToResults("model", "model.root.System2.*")
+oms_addSignalsToResults("model", "model.root.Gain.*")
+
+oms_removeSignalsFromResults("model", "model.root.System2.parameter_1")
+oms_removeSignalsFromResults("model", "model.root.Gain.y")
+
+oms_export("model", "signalFilter.ssp")
+
+oms_terminate("model")
+oms_delete("model")
+
+oms_importFile("signalFilter.ssp")
+
+src, status = oms_exportSnapshot("model")
+print(src)
+
+oms_instantiate("model")
+
+oms_setStartTime("model", 0.0)
+oms_setStopTime("model", 0.0)
+oms_setResultFile("model", "model_res.csv")
+
+oms_initialize("model")
+readFile("model_res.csv")
+
+oms_terminate("model")
+oms_delete("model")
+
+-- Result:
+-- <?xml version="1.0"?>
+-- <oms:snapshot>
+-- 	<oms:ssd_file name="SystemStructure.ssd">
```

Comment:
We should simplify the snapshot by replacing `oms:ssd_file` etc. with just `oms:file` for all files.